mmengine/runner/checkpoint.py

```diff
@@ -252,8 +252,10 @@ def load_checkpoint(cls, filename, map_location=None, logger=None):
 
         checkpoint_loader = cls._get_checkpoint_loader(filename)
         class_name = checkpoint_loader.__name__
-        print_log(f'{class_name[10:]} loads checkpoint from path: {filename}',
-                  logger)
+        print_log(
+            f'Loads checkpoint by {class_name[10:]} backend from path: '
+            f'{filename}',
+            logger='current')
```

Comment:
Use 'current' logger will make the input logger useless. Is it intended? Maybe this will lead to confusion.